apps/examples/client/example_channel_view.coffee

```diff
@@ -0,0 +1,12 @@
+Backbone = require 'backbone'
+{ mountWithApolloProvider } = require '../../../react/apollo/index.js'
+{ default: FollowButton } = require '../../../react/components/FollowButton'
+
+module.exports = class ExampleChannelView extends Backbone.View
+
+  initialize: ->
+    console.log('initializing', @$el.data('id'))
+    @initializeFollowButton()
+
+  initializeFollowButton: ->
+    mountWithApolloProvider FollowButton, { id: @$el.data('id'), type: 'CHANNEL' }, $('.js-example-channel-follow')
```

Comment:
Avoid using views unless they are directly responding to a model. Otherwise just use jQuery and scope the els appropriately.